### ISSUE TITLE
Error starting ASIO (Win 10)

### ISSUE THREAD

**Issue by tilllt** [eiz/SynchronousAudioRouter#44]
When i start reaper with SAR installed, the only thing i see is "Error starting ASIO"

![screenshot_1](https://user-images.githubusercontent.com/1854364/36475733-ef73a550-16fb-11e8-9433-6a03aaeba0a7.png)

I am using ASIO4ALL as my "hardware" ASIO for now.

I uninstalled and reinstalled SAR but that didnt help anything. Also, starting Reaper as Admin or normal user doesnt change a thing...

Any Clues?

**Comment by rehlinger** [comment]
Same issue. I've tried both ASIO4ALL and UMC ASIO hardware interfaces and it fails to start either. New user though so not sure. Also, JackAudio fails to connect and start the driver.

**Comment by tilllt** [comment]
Ok, i think i found the problem, but i really dont feel comfortable to completely disable integrity check for device drivers just to install SAR, is there another way?

![screenshot_27](https://user-images.githubusercontent.com/1854364/36540442-b58f0c6e-17da-11e8-9e4c-6e0a63469b85.png)
![screenshot_28](https://user-images.githubusercontent.com/1854364/36540444-b5ba711a-17da-11e8-9776-60d5bf98d5a3.png)

**Comment by rehlinger** [comment]
Yes, this is the problem. Restarting windows with advance settings to disable signature enforcement temporarily causes the driver to work with reaper.

**Comment by JonnyLong** [comment]
I was getting this same issue for a long time with my notebook, and was disabling signature enforcement to get around it. On a hunch, I tried going into my BIOS settings and disabling secure boot and somehow that fixed it. I now can use SAR regularly without using advanced boot settings to disable signature enforcement. Hope this helps!

**Comment by bit-drifter** [comment]
I think Jonny is correct as there is a note in the 4.50 changelog for Virtual Audio Cable that says  "Signed with EV certificate to allow loading by Win10 in Secure Boot mode."

More information is here 
https://blogs.msdn.microsoft.com/windows_hardware_certification/2016/07/26/driver-signing-changes-in-windows-10-version-1607/

**Comment by dooitze** [comment]
In the new build 19041.208 of Version 2004 the problem has reappeared, with the same error in the driver tab.

**Comment by jeinstei** [comment]
eiz -- I completely agree with not paying a toll, but I would love to get this working without having such problems. Would you take a donation for a Certum signing cert? https://www.certum.eu/en/cert_offer_en_open_source_cs/

**Comment by amurzeau** [comment]
The open source certificate is not what's required for a driver. A driver requires an EV certificate now with Windows 10 IIRC, and that's 360€ for only one year. And there are probably heavy administrative stuff to handle with this type of certificate I guess.

**Comment by jeinstei** [comment]
Ouch. Understood. They really need a program where people can at least sign drivers on their local machine somehow. What an opaque process for users of open source and developers.
